src/fastoad/models/performances/mission.py

```diff
@@ -0,0 +1,252 @@
+"""
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [One-line docstring should fit on one line with quotes (found 3) (D200)](https://app.codacy.com/gh/fast-aircraft-design/FAST-OAD/pullRequest?prid=5618066)